docs/sources/alerting/fundamentals/notifications/group-alert-notifications.md

```diff
@@ -69,22 +69,117 @@ If you want to receive every alert as a separate notification, you can do so by
 
 ## Timing options
 
-The timing options decide how often notifications are sent for each group of alerts. There are three timers that you need to know about: Group wait, Group interval, and Repeat interval.
+In the notification policy, you can also configure how often notifications are sent for each [group of alerts](#group-notifications). There are three distinct timers applied to all the resulting groups within the notification policy:
 
-#### Group wait
+- **[Group wait](#group-wait)**: the time to wait before sending the first notification for a new group of alerts.
+- **[Group interval](#group-interval)**: the time to wait before sending a notification about changes in the alert group.
+- **[Repeat interval](#repeat-interval)**: the time to wait before sending a notification if the group has not changed since the last notification.
 
-Group wait is the amount of time Grafana waits before sending the first notification for a new group of alerts. The longer Group wait is the more time you have for other alerts to arrive. The shorter Group wait is the earlier the first notification is sent, but at the risk of sending incomplete notifications. You should always choose a Group wait that makes the most sense for your use case.
+These timers introduce delays in delivering notifications, enabling the grouping of incoming alerts into one notification to reduce the number of notifications.
```

Comment:
```suggestion
These timers reduce the number of notifications sent. By delaying the delivery of notifications, incoming alerts can be grouped into just one notification instead of many.
```